AirCasting/SDSync/ViewsFlow/ClearingSDCard/ClearingSDCardViewModel.swift

```diff
@@ -1,18 +1,95 @@
 // Created by Lunar on 06/12/2021.
 //
 import Foundation
+import CoreBluetooth
+
+enum ClearingSDCardError {
+    case undefined
+    case noConnection
+    case noClearing
+}
 
 protocol ClearingSDCardViewModel: ObservableObject {
+    var isClearingCompleted: Published<Bool>.Publisher { get }
+    var shouldDismiss: Published<Bool>.Publisher { get }
     var presentNextScreen: Bool { get set }
     var isSDClearProcess: Bool { get set }
+    func clearSDCard()
```

Comment:
like I wrote in Janek's PR - viewmodel should not be asked to do a certain thing by the view. It should only be informed what happend view-wise (like user tapped something, switched a switch, stuff like that). So for example here it should be phrased more like `clearSDCardButtonTapped()` or smth like that.

Reply:
As long as it is used in the .onAppear modifier I am really not sure @pawelgil. Does `clearSDCardButtonTapped()` still work here?